src/core/network/createNetwork.js

```diff
@@ -36,71 +36,65 @@ export default (config, logger, lifecycle, networkStrategy) => {
 
   return {
     /**
-     * The object returned from network.newRequest
-     *
-     * @typedef {Object} Request
-     * @property {Function} send Call this function when you are ready to send
-     * the payload. Returns a promise yielding the raw response body.
-     * @property {Object} payload Payload object that will be sent.
+     * Create a new payload.  Once you have added data to the payload, send it with
+     * the sendRequest method.
      */
+    createPayload() {
+      return createPayload();
+    },
```

Comment:
I think you could just do:

```
createPayload,
```

instead of:

```
createPayload() {
  return createPayload();
},
```

Reply:
Indeed! Old Java habits die hard